frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -916,30 +933,51 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
                                     label: <span>Multiple variants with rollout percentages (A/B/n test)</span>,
                                     value: 'multivariate',
                                 },
+                                {
+                                    label: <span>Remote config (single payload)</span>,
+                                    value: 'remote_config',
+                                    disabledReason:
+                                        featureFlag.experiment_set && featureFlag.experiment_set?.length > 0
+                                            ? 'This feature flag is associated with an experiment.'
+                                            : undefined,
+                                },
                             ]}
                             onChange={(value) => {
-                                if (value === 'boolean' && nonEmptyVariants.length) {
+                                if (['boolean', 'remote_config'].includes(value) && nonEmptyVariants.length) {
                                     confirmRevertMultivariateEnabled()
                                 } else {
                                     setMultivariateEnabled(value === 'multivariate')
+                                    setRemoteConfigEnabled(value === 'remote_config')
                                     focusVariantKeyField(0)
                                 }
                             }}
-                            value={multivariateEnabled ? 'multivariate' : 'boolean'}
+                            value={flagType}
                         />
                     </div>
                     <div className="text-muted mb-4">
-                        {capitalizeFirstLetter(aggregationTargetName)} will be served{' '}
-                        {multivariateEnabled ? (
+                        {featureFlag.is_remote_configuration ? (
+                            <span>
+                                Remote config flags provide runtime configuration values in your app. Read more in the{' '}
+                                <Link to="https://posthog.com/docs/feature-flags/remote-config" className="italic">
```

Comment:
```suggestion
                                <Link to="https://posthog.com/docs/feature-flags/remote-config">
```
(links in our feature flag page aren't italic 😄 )